pysparkle/pysparkle-pipeline.yml

```diff
@@ -0,0 +1,278 @@
+              poetry run pytest -rf -v tests -k unit
+            displayName: 'Run Unit Tests'
+            workingDirectory:  '$(self_repo_pysparkle_dir)'          
+          - bash: |
+              poetry build
+            displayName: Build Project
+            workingDirectory: '$(self_repo_pysparkle_dir)/pysparkle'
+          - task: CopyFiles@2
+            displayName: Copy Whl
+            inputs:
+              SourceFolder: '$(self_repo_pysparkle_dir)/dist/'
+              Contents: '*.whl'
+              TargetFolder: '$(Build.ArtifactStagingDirectory)/pysparklewheel'
+          - task: PublishBuildArtifacts@1
+            displayName: 'Publish Artifact: pysparklewheel'
+            inputs:
+              ArtifactName: pysparklewheel
+              PathtoPublish: '$(Build.ArtifactStagingDirectory)/pysparklewheel'
+
+#############################################################
+# Deploy to non Prod
+#############################################################
+  - stage: Deploy_NonPROD_Stage
+    variables:
+        - group: amido-stacks-de-pipeline-nonprod
+        - group: amido-stacks-infra-credentials-nonprod
+        - name: version_number
+          value: "$(version_major).$(version_minor).$(version_revision)"
+    dependsOn: Build_Stage
+    jobs:
+      - deployment: Deploy_NonPROD
+        displayName: 'Deploy To NonPROD'
+        environment: ${{ variables.domain }}-nonprod
+        pool:
+          name: $(agentpool_name)
+        strategy:
+          runOnce:
+            deploy:  
+              steps:
+              - task: DownloadPipelineArtifact@2
+                displayName: Download Build Artifacts
+                inputs:
+                  targetPath: '$(System.DefaultWorkingDirectory)'
+              - script: dir
+                displayName: List Artifact in Workspace
+                workingDirectory: '$(System.DefaultWorkingDirectory)/pysparklewheel'
+              # - script: pip install databricks-cli
+              #   displayName: Install Databricks CLI
+              - task: AzureKeyVault@2
+                inputs:
+                    azureSubscription: 'Stacks.Pipeline.Builds'
+                    KeyVaultName: $(keyvault_name)
+                    SecretsFilter: '*'
+                    RunAsPreJob: false
+                displayName: 'Get Keyvault Secrets' 
+              - script: |
+                          echo "$(databricks-host)
+                          $(databricks-token)" | databricks configure --token
+                displayName: Configure Databricks-cli
+              - script: |
+                        echo Destination is /FileStore/jars
+                        echo Source is $(System.DefaultWorkingDirectory)/pysparklewheel/
+                        databricks fs cp -r --overwrite $(System.DefaultWorkingDirectory)/pysparklewheel/*.whl dbfs:/FileStore/jars/pysparkle-latest-py3-none-any.whl
+                displayName: Upload whl
+
+    # #   # Start Testing 
```

Comment:
Why commented?

Reply:
These will be required when we include end to end testing in future, Unit testing is happening in build process.

Reply:
Yeah, I'm just not a fan of keeping lots of commented code, especially that I think most of it won't be needed for e2e, but I'm not insisting - resolving :)

Reply:
No worries, I have removed it for now :)